src/fetch/dune.py

```diff
@@ -163,24 +162,6 @@ def get_vouches(self) -> list[DuneRecord]:
             )
         )
 
-    def get_period_totals(self) -> PeriodTotals:
```

Comment:
Out of curiosity: What's the reason we removed this?

Reply:
It is not used for anything (i.e. dead code).